app/controllers/api/v2/form_documents_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
 
     documents = Api::V2::FormDocument.all
     documents = documents.where(tag:) if tag.present?
-    paginated = documents.page(params[:page]).per(params[:per_page] || DEFAULT_PAGE_SIZE)
+    paginated = documents.page(params[:page]).per(params[:per_page] || DEFAULT_PAGE_SIZE).order(:id)
```

Comment:
I think it would make sense to update the commit name to `Order returned form documents by id` to more accurately describe what this is doing

Reply:
Good spot, I changed the ordering but forgot to update the message! 🤦‍♂️

Reply:
done